hedera-node/src/main/java/com/hedera/services/context/properties/GlobalDynamicProperties.java

```diff
@@ -87,6 +88,7 @@ public void reload() {
 		shouldKeepRecordsInState = properties.getBooleanProperty("ledger.keepRecordsInState");
 		maxNFTMetadataBytes = properties.getIntProperty("tokens.nfts.maxMetadataBytes");
 		maxBatchSizeBurn = properties.getIntProperty("tokens.nfts.maxBatchSizeBurn");
+		maxBatchSizeTransfer = properties.getIntProperty("tokens.nfts.maxBatchSizeTransfer");
```

Comment:
Currently we are using:
```Java
maxTransfersLen = properties.getIntProperty("ledger.transfers.maxLen");
maxTokenTransfersLen = properties.getIntProperty("ledger.tokenTransfers.maxLen");
```

I think that it would be more consistent if we plug our transfer size there. Suggestion:
```Java
maxNftTransfersLen = properties.getIntProperty("ledger.nftTransfers.maxLen");
```